client/settings/payment-request-section/__tests__/index.test.js

```diff
@@ -3,9 +3,31 @@ import { screen, render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 import PaymentRequestSection from '..';
+import {
+	usePaymentRequestEnabledSettings
+} from '../../../data/index';
+
+jest.mock( '../../../data/index', () => ( {
+	usePaymentRequestEnabledSettings: jest.fn(),
+} ) );
+
+const getMockPaymentRequestEnabledSettings = (
+	isEnabled,
+	updateIsPaymentRequestEnabledHandler
+) => [ isEnabled, updateIsPaymentRequestEnabledHandler ];
 
 describe( 'PaymentRequestSection', () => {
+	beforeEach( () => {
+		usePaymentRequestEnabledSettings.mockReturnValue(
+			getMockPaymentRequestEnabledSettings( false, jest.fn() )
+		);
+	} );
+
 	it( 'should enable express checkout locations when express checkout is enabled', () => {
+		usePaymentRequestEnabledSettings.mockReturnValue(
+			getMockPaymentRequestEnabledSettings( true, jest.fn() )
```

Comment:
(minor) what are your thoughts on just using the raw values for the tests (without the need for a `getMockPaymentRequestEnabledSettings` that is "building" an array with the values)?

```suggestion
		[ true, jest.fn() ]
```

Reply:
Yes! :rocket: